plugins/frontend/backstage-plugin-iframe/src/plugin.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import {
+  createComponentExtension,
+  createPlugin,
+  createRouteRef,
+  createRoutableExtension,
+} from '@backstage/core-plugin-api';
+
+export const entityContentRouteRef = createRouteRef({
+  id: 'iframe',
+});
+
+export const iframePlugin = createPlugin({
+  id: 'iframe',
+  routes: {
+    entityContent: entityContentRouteRef,
+  },
+});
+
+export const EntityIFrameCard = iframePlugin.provide(
+  createComponentExtension({
+    name: 'EntityIFrameCard',
+    component: {
+      lazy: () =>
+        import('./components/IFrameComponent').then(
+          m => m.IFrameCard,
+        ),
+    },
+  }),
+);
+
+export const EntityIFramePage = iframePlugin.provide(
```

Comment:
This is using 'content' routeref. Should this be a Content type instead of a Page?